int64 player_id
---
int64 balance
